client/solve.go

```diff
@@ -205,11 +207,18 @@ func (c *Client) solve(ctx context.Context, def *llb.Definition, runGateway runG
 			frontendInputs[key] = def.ToPB()
 		}
 
+		exporters := make([]*controlapi.Exporter, 0, len(opt.Exports))
+		for _, exp := range opt.Exports {
+			exporters = append(exporters, &controlapi.Exporter{
+				Name:  exp.Type,
+				Attrs: exp.Attrs,
+			})
+		}
+
 		resp, err := c.controlClient().Solve(ctx, &controlapi.SolveRequest{
 			Ref:            ref,
 			Definition:     pbd,
-			Exporter:       ex.Type,
-			ExporterAttrs:  ex.Attrs,
+			Exporters:      exporters,
```

Comment:
We should still set the deprecated field so that old daemons continue to work. Ideally, we would also check if daemon supports multiple exporters or not and error if the client required multiple but daemon didn't support it.